bench/wtperf/config.c

```diff
@@ -734,13 +736,13 @@ config_to_file(CONFIG *cfg)
 	fp = NULL;
 
 	/* Backup the config */
-	req_len = strlen(cfg->home) + 100;
-	if ((path = calloc(req_len, 1)) == NULL) {
+	req_len = strlen(cfg->home) + 14;
+	if ((path = calloc(req_len + 1, 1)) == NULL) {
 		(void)enomem(cfg);
 		goto err;
 	}
 
-	snprintf(path, req_len + 14, "%s/CONFIG.wtperf", cfg->home);
+	snprintf(path, req_len, "%s/CONFIG.wtperf", cfg->home);
```

Comment:
@daveh86, a safer way to do this is:
```
req_len = strlen(cfg->home) + strlen("/CONFIG.wtperf") + 1;
```
that way you don't get caught if the name changes, or you just miscounted the characters (which I invariably do).

On modern compilers, it's just as fast as calling `sizeof`, too, the compiler figures out it's a `strlen` of a constant string in most cases (and we don't care here, anyway).